lib/commons/rscommons/download_dem.py

```diff
@@ -160,22 +169,21 @@ def verify_areas(raster_path, boundary_shp):
     # We could just use Rasterio's CRS object but it doesn't seem to play nice with GDAL so....
     raster_ds = gdal.Open(raster_path)
     raster_srs = osr.SpatialReference(wkt=raster_ds.GetProjection())
+    log.debug(f'Raster Spatial Ref is {raster_srs.GetName()} and linear units are {raster_srs.GetLinearUnitsName()}')
+
+    with get_shp_or_gpkg(boundary_shp) as layer:
```

Comment:
This looks safe to me. We should see it run on a regular rscontext project though.